cmd/kaniko-docker/main.go

```diff
@@ -146,9 +163,26 @@ func run(c *cli.Context) error {
 			EnableCache:   c.Bool("enable-cache"),
 			CacheRepo:     c.String("cache-repo"),
 			CacheTTL:      c.Int("cache-ttl"),
+			DigestFile:    digestFileName,
 		},
 	}
-	return plugin.Exec()
+	err = plugin.Exec()
+	if err != nil {
+		return err
+	}
+
+	if c.String("output-file") != "" {
```

Comment:
This is a common code which needs to be present in kaniko.go file

Reply:
to me it looks like kaniko.go is just an abstraction of kaniko executor, so it feels that this code belongs here